### PR TITLE
[UI] Add Logged in User indicator 

### PR DESCRIPTION
**Pull request by johnkuney**
Related to #826

Description
---
Adding the user email to the top right of the screen across the pages to indicate who's logged in.
I also added a few minor UI tweaks and updated twitter to X 

As far as I can tell, there is not a way to get the email at the component level, so I had to add it to the getStaticProps for every page. But if there is a cleaner way to do this let me know @chedieck  

Test plan
---
Preview the site see the email appears on all the main pages. It should also link to /account

### CODE REVIEW

**Review by Klakurka — CHANGES_REQUESTED** [review]
- I think we can skip the account icon for now.
- Maybe we can make it styled more like the main nav buttons where hovering over it will change the bg of the button; on-hover underlining removed.
- I think it would look better if it was positioned at the top of the div like the page's <h2> is, rather then being postioned vertically centered.

**Comment by johnkuney** [comment]
For sure, added

**Review by Klakurka — APPROVED** [review]

**Review by chedieck — CHANGES_REQUESTED** [review]
Left a suggestion and some code readability remarks

File: pages/wallets/index.tsx [hunk 1]
```diff
@@ -25,13 +28,22 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
...
+    props: {
+      userId,
+      user
+    }
   }
 }
 
 interface WalletsProps {
   userId: string
+  user: UserWithSupertokens
```
[the remark this answers is not in the document]
Comment (johnkuney) [reply pages/wallets/index.tsx:44]: 'this.props.user.userProfile.id' right?

**Comment by johnkuney** [comment]
Cool, added those changes

File: pages/wallets/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (chedieck) [reply pages/wallets/index.tsx:44]: yep, indeed

**Review by chedieck — APPROVED** [review]
LGTM!

**Review by Klakurka — APPROVED** [review]

**Pull request merged**